userspace/libsinsp/container.h

```diff
@@ -127,6 +133,9 @@ class sinsp_container_info
 	string m_name;
 	string m_image;
 	string m_imageid;
+	string m_imagerepo;
```

Comment:
Do you want to make these "N/A" or similar, instead of blank, in the case where it can't be parsed? Thinking particularly about parsing old capture files where the container json event is missing these fields.

Reply:
Sure, I'll create a default constructor that assign "<N/A>" to all the string fields.

Reply:
Actually, sysdig already print out `<N/A>` if string fields are empty, isn't it enough?

Reply:
Oh ok, that works then.